ui-src/src/scoring.ts

```diff
@@ -42,7 +42,7 @@ export const getScores = (gameBoard: GameBoard, actions: Action[]): Map<Hash, nu
 // only the first action on each square is allowed to generate a score.
 // Assumes actions are sorted ascending in time
 export const isFirst = (action: Action, index: number, actions: Action[]): boolean => {
-  if (action.actionType === "chat") { return false; }
+  if (action.actionType === "chat") { return true; }
```

Comment:
This is the most crucial fix. @JettTech and I tested and it seems to still work. Could use your eyes @willemolding if you get a chance.